doc/site/core/fn.markdown

```diff
@@ -30,4 +30,13 @@ The number of arguments the function requires.
 
 ### **call**(args...)
 
-**TODO**
+Invokes the function with the given arguments.
+
+    :::dart
+    var fn = new Fn { |arg|
+      IO.print(arg)
+    }
+    fn.call("Hello world") // Hello world.
+
+It is a runtime error if the number of arguments given does not equal the arity
+of the function.
```

Comment:
You can pass extra arguments and they will be discarded. It's only a runtime error if there are too *few* arguments.